.eslintrc

```diff
@@ -85,7 +85,7 @@
     "react/jsx-no-bind": ["error"],
     "react/jsx-no-comment-textnodes": ["error"],
     "react/jsx-no-duplicate-props": ["error"],
-    "react/jsx-no-literals": ["error"],
```

Comment:
Do we need to change eslint rules to update new version of material-ui ? I am not clear about this change.

Reply:
I was getting an error, due to the ForwardRef I added in some components ... it was teelling me that my functional components should be named...something like Link.name = 'Link' what I find meaningless.